src/pdfs/ShippingLabels/ShippingLabelsPdf.tsx

```diff
@@ -68,29 +72,33 @@ const ParcelCard: React.FC<{ datum: ParcelClients }> = ({ datum }) => {
                     {datum.collection_datetime} | {datum.collection_centre}
                 </Text>
                 <Text style={[styles.col, { textAlign: "right" }]}>
-                    {datum.index}/{datum.total}
+                    {index}/{total}
                 </Text>
             </View>
         </View>
     );
 };
 
-interface ShippingLabelsPdfProps {
-    data: ParcelClients[][];
+export interface ShippingLabelsPdfProps {
+    data: ParcelClients[];
 }
 
 const ShippingLabelsPdf: React.FC<ShippingLabelsPdfProps> = ({ data }) => {
+    const dataLength = data.length;
```

Comment:
Is this really needed as a const? feels like it could just be put directly into the props